nemo/collections/asr/models/rnnt_models.py

```diff
@@ -159,7 +159,11 @@ def extract_rnnt_loss_cfg(self, cfg: Optional[DictConfig]):
 
     @torch.no_grad()
     def transcribe(
-        self, paths2audio_files: List[str], batch_size: int = 4, return_hypotheses: bool = False
+        self,
+        paths2audio_files: List[str],
+        batch_size: int = 4,
+        logprobs: bool = False,
```

Comment:
Lets remove the `logprobs` option here - RNNT cant compute log probs easily during inference.

Reply:
Done.

Reply:
done